demo/html5canvas/testbed.html

```diff
@@ -21,6 +21,9 @@
         <script type="text/javascript">  
             var Box2D;
             if (!Box2D) Box2D = (typeof Box2D !== 'undefined' ? Box2D : null) || Module;
+            var Box2DConstructor = Box2D
+            Box2D = Box2D()
+            Module = Box2D
```

Comment:
this also doesn't look used?

Reply:
I removed line 24 (`Box2DConstructor`) but if I remove the other 2 lines, I get this error message:
```
Uncaught TypeError: Box2D.JSDraw is not a constructor
    at getCanvasDebugDraw (embox2d-html5canvas-debugDraw.js:85)
    at init (embox2d-html5canvas-testbed.js:275)
    at window.onload (testbed.html:27)
```
I think those 2 lines are essential to the working of the test bed.